ci/Dockerfile-envoy-alpine-debug

```diff
@@ -1,3 +1,17 @@
 FROM frolvlad/alpine-glibc
 
+ENV loglevel=info
```

Comment:
I think this should be debug

Reply:
i forgot to mention, this can be overridden at runtime, `docker run -e "loglevel=debug" envoyproxy/envoy`

Reply:
Yes, debug logging is pretty expensive. Ok that sounds good to me